airflow-core/src/airflow/ui/openapi-merge.json

```diff
@@ -0,0 +1,11 @@
+{
+  "inputs": [
+    {
+      "inputFile": "../api_fastapi/core_api/openapi/v1-rest-api-generated.yaml"
```

Comment:
I wonder if we should rename this to v2 to match the rest api versioning?

Reply:
I had the exact same thought, I'll do that in a separate PR I think though

Reply:
Or maybve v3 :D